config/locales/views.en.yml

```diff
@@ -438,6 +433,9 @@ en:
         actions: Actions
       button:
         new: New Volunteer
+    send_reminder_button:
+      send_reminder: Send Reminder
+      tooltip: Sends to volunteer a reminder email to input case contacts
```

Comment:
I know you didn't write this but that could be phrased more clearly.

Reply:
Agreed, it would be nice to change this up while you're in here